src/useCases/ApproveFeedService.spec.js

```diff
@@ -0,0 +1,73 @@
+const ApproveFeedService = require("./ApproveFeedService");
+
+describe("ApproveFeedService", () => {
+  let approveFeedService;
+
+  const feedRepository = {
+    getFeedById: jest.fn(),
+    updateIsPendentByIdAndIsPendent: jest.fn(),
+  };
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const mockFeed = [
+    {
+      id: "1",
+      title: "Feed de member",
+      image: "http://localhost:3000/image.png",
+      content: "Criado com member",
+      created_at: "2024-06-25T14:05:07.627Z",
+      is_pendent: true,
+      user_id: "1",
+    },
+  ];
+
+  it("Should to throw NotFoundError if not exist feed to approve", async () => {
+    try {
+      const params = {
+        id: 1,
+        is_pendent: false,
+      };
+
+      feedRepository.getFeedById.mockResolvedValue([]);
+
+      approveFeedService = new ApproveFeedService(feedRepository);
+
+      await approveFeedService.execute(params);
+    } catch (err) {
+      expect(err.message).toEqual("Feed inexistente.");
+    }
+  });
+
+  it("Should return feed by id", async () => {
```

Comment:
@gumeeee creio que seria interessante mudar o nome do teste da linha 44 para algo que faça sentido com a funcionalidade implementada. Exemplo: **Should approve feed by id**